stix_shifter/stix_translation/src/modules/elastic_ecs/query_constructor.py

```diff
@@ -0,0 +1,342 @@
+from stix_shifter.stix_translation.src.patterns.pattern_objects import ObservationExpression, ComparisonExpression, \
+    ComparisonExpressionOperators, ComparisonComparators, Pattern, \
+    CombinedComparisonExpression, CombinedObservationExpression, ObservationOperators
+from stix_shifter.stix_translation.src.transformers import TimestampToMilliseconds, DateTimeToUnixTimestamp, EpochSecondsToTimestamp
+from stix_shifter.stix_translation.src.json_to_stix import observable
+import datetime
+import logging
+import re
+
+
+DEFAULT_LIMIT = 10000
```

Comment:
A result limit of 10000 is already defined in stix_translation.py so it doesn't need to be here.

Reply:
Yes, I will remove this line.

Reply:
Updated code to remove result limit from query_constructor. Ref: c6aee8a
Added DEFAULT_LIMIT in transmission module. For Elasticsearch, the result limit is added in transmission module based on length parameter or DEFAULT_LIMIT (10000). Please review and let me know your comments.